build/common/config/.golangci.yml

```diff
@@ -72,9 +88,9 @@ linters-settings:
   lll:
     # max line length, lines longer will be reported. Default is 120.
     # '\t' is counted as 1 character by default, and can be changed with the tab-width option
-    line-length: 160
+    line-length: 120
     # tab width in spaces. Default to 1.
-    tab-width: 1
+    tab-width: 4
```

Comment:
Taking the initiative on the controversial topics, I see 😄 

This seems fine to me though.